NEWS

```diff
@@ -1,5 +1,5 @@
-Flocker 0.4.0pre1 (2015-03-24)
-==============================
+Flocker 0.4.0 (2015-04-07)
+==========================
 
 - New getting started guide. (FLOC-1261)
```

Comment:
This was already include in `0.3.2+doc1`, so I'm not sure if it worth mentioning here.